appcues/src/main/java/com/appcues/SessionMonitor.kt

```diff
@@ -43,46 +41,23 @@ internal class SessionMonitor(
         }
     }
 
-    fun start() {
-        if (storage.userId.isEmpty()) return
+    fun startNewSession(): Boolean {
+        if (storage.userId.isEmpty()) return false
         _sessionId = UUID.randomUUID()
-        analyticsTracker.track(AnalyticsEvent.SessionStarted, null, true)
+        updateLastActivity()
+        return true
     }
 
-    fun reset() {
-        // this is interactive: true since a reset should flush to network immediately (with previous user ID)
-        // and the next session start will be sent in a new request, with the new user ID
-        analyticsTracker.track(AnalyticsEvent.SessionReset, null, true)
-        _sessionId = null
-    }
-
-    fun checkSession(errorMessage: String): Boolean {
-        if (isActive) return true
-
-        logcues.info("No active session - $errorMessage")
```

Comment:
logging these "no active session" messages seemed minimal value IMO. I don't feel strongly though and can re-introduce some logging if desired.